src/ddmd/expression.d

```diff
@@ -7823,6 +7823,23 @@ extern (C++) class UnaExp : Expression
         return null;
     }
 
+    final Expression incompatibleType()
```

Comment:
I see this spelled as `incompatibleTypes()` (with a trailing `s`) in other places. This looks wrong.